bootstrap4/.js/src/main/scala/io/udash/bootstrap/nav/UdashNavbar.scala

```diff
@@ -87,6 +86,39 @@ object UdashNavbar {
   )(
     navigationFactory: Binding.NestedInterceptor => UdashNav[ItemType, ElemType],
     brand: Modifier = ()
+  ): UdashNavbar[ItemType, ElemType] = {
+    import io.udash.css.CssView._
+    new UdashNavbar(expandBreakpoint, darkStyle, backgroundStyle, position, componentId)(
+      interceptor => navigationFactory(interceptor).render.styles(BootstrapStyles.NavigationBar.nav),
+      brand
+    )
+  }
+
+  /**
+    * Creates a default, responsive navigation bar.
+    * More: <a href="http://getbootstrap.com/docs/4.1/components/navbar/">Bootstrap Docs</a>.
+    *
+    * @param expandBreakpoint  Screen size breakpoint to switch between collapsed and expanded menu.
+    * @param darkStyle         If true, enables dark navigation bar theme.
+    * @param backgroundStyle   Selects navigation bar background style.
+    * @param position          Sets bar position.
+    * @param componentId       An id of the root DOM node.
+    * @param navigationFactory Navigation content factory - if you want to clean up the created navigation on
+    *                          the navigation bar cleanup pass it to the provided interceptor.
+    * @param brand             A brand DOM element.
```

Comment:
it's rather a `Modifier`

Reply:
The "element" here means something you want to use in a navigation bar, not an instance of `Element` type.

Reply:
Ok.